starter-core/src/main/java/io/micronaut/starter/feature/function/template/http/httpFunctionKotlinController.rocker.raw

```diff
@@ -0,0 +1,42 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+}
+
+
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.HttpStatus
+
+@@Controller("/")
```

Comment:
You don't need to qualify the controller. that it is to say: 

You can just use `@Controller` 

instead of `@Controller("/")`

Reply:
I've done this change.